public/index.html

```diff
@@ -3,8 +3,12 @@
   <head>
     <meta charset="utf-8" />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;700&family=Merriweather:wght@400&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
vet ikke hva som er best av dette eller å lagra fonter i prosjektet som vi gjorde før...

Reply:
den urlen der kan jo endre seg i fremtiden da, så er sikkert tryggest å lagre fonter lokalt

Reply:
de anbefaler faktisk å importere slik, men sier ikke noe om hvorfor... https://developers.google.com/fonts/faq#should_i_host_fonts_on_my_own_website%E2%80%99s_server
Blir litt ekstra kode om vi skal lagre lokalt også, ref det som er fjernet i PRen (+ litt ekstra for de vi ikke har atm)